src/components/NavHeader/NavHeader.styled.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+export const NavHeaderWrapperFull = styled.section`
+  position: relative;
+  width: 100%;
+  top: ${(props) => props.theme.spacing.zero};
+  background-color: ${(props) => props.theme.color.primary};
+  padding: ${(props) => props.theme.spacing.xSmall} ${(props) => props.theme.spacing.zero};
```

Comment:
```suggestion
${({ theme }) => css`
  position: relative;
  width: 100%;
  top: ${theme.spacing.zero};
  background-color: ${theme.color.primary};
  padding: ${theme.spacing.xSmall} ${theme.spacing.zero};
`}
```

just like we do in sbf, you can simplify the code by using a single funcion